java/memory/src/main/java/io/netty/buffer/PooledByteBufAllocatorL.java

```diff
@@ -70,13 +61,56 @@ public int getChunkSize() {
     return allocator.chunkSize;
   }
 
-  private class InnerAllocator extends PooledByteBufAllocator {
+  public long getHugeBufferSize() {
+    return hugeBufferSize.get();
+  }
+
+  public long getHugeBufferCount() {
+    return hugeBufferCount.get();
+  }
+
+  public long getNormalBufferSize() {
+    return normalBufferSize.get();
+  }
+
+  public long getNormalBufferCount() {
+    return normalBufferSize.get();
+  }
+
+  private static class AccountedUnsafeDirectLittleEndian extends UnsafeDirectLittleEndian {
+    private final long initialCapacity;
+    private final AtomicLong count;
+    private final AtomicLong size;
 
+    private AccountedUnsafeDirectLittleEndian(LargeBuffer buf, AtomicLong count, AtomicLong size) {
```

Comment:
Am I missing an override for copy (like it was done in LargeBuffer previously)? The thing is that UnsafeDirectLittleEndian had not such override. Also, since this is a copy, should we then increment the counters at copy time (not done either previously).